mikidown/mikiwindow.py

```diff
@@ -1264,7 +1264,15 @@ def closeEvent(self, event):
                           top-level widgets
             saveState: Restores the state of this mainwindow's toolbars
                        and dockwidgets
+            BUG: here cos we want a quit or minimize to tray and a popup
+                 we dont wanna crash the application with out warning and WTF atmo
```

Comment:
Can you list the steps you did to create the crash? I remember trying to reproduce this crash while I was working on the updates for the mikidown tray, but I couldn't manage to reproduce it again.

Once you have a reliable set of instructions, can you update #75 with the instructions to reproduce it? That'd be helpful.